message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1249,6 +1273,19 @@
       <field type="float" name="I">I component</field>
       <field type="float" name="D">D component</field>
     </message>
+    <message id="195" name="DEEPSTALL">
```

Comment:
Due to my use case not having MAVLink2 support yet. It is on the agenda, but discussions with @tridge we reached the conclusion that this could go within the MAVLink1 ns, given that there are still 21 unallocated Ardupilot specific id's.